docs/source/pipelines.rst

```diff
@@ -118,38 +119,106 @@ Key concepts
 
 Usage
 -----
+Development environments
+~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend to develop models with MLflow Pipelines with one of the following environment settings:
 
-The general workflow for using MLflow Pipelines is as follows:
+[**Databricks**]
+  - Edit YAML config and Python files in Databricks Repos. Open separate browser tabs for each
+    file module that you wish to modify.
+  - Use ``notebooks/databricks.py`` as the driver to run pipeline steps and inspect its output.
+  - Pin the workspace browser for easy file navigation.
+
+  .. image:: _static/images/pipelines_databricks_ui.png
```

Comment:
Very nitty, but can we edit the screenshot so that it doesn't show "E2 Dogfood" with a doge icon?

Reply:
Will change it to DB logo.

Reply:
Done.